pkg/interactive/validation.go

```diff
@@ -226,7 +226,8 @@ func RegExpBoolean(restr string) Validator {
 
 // SubnetsCountValidator get a slice of `[]core.OptionAnswer` as an interface.
 // e.g. core.OptionAnswer { Value: subnet-04f67939f44a97dbe (us-west-2b), Index: 0 }
-func SubnetsValidator(awsClient aws.Client, multiAZ bool, privateLink bool, hostedCP bool) Validator {
+func SubnetsValidator(awsClient aws.Client, multiAZ bool, privateValue bool, hostedCP bool,
```

Comment:
This is not a great method in terms of parameters :melting_face:

Reply:
yeahhhhh lol. im not happy with it either